sceptre/template.py

```diff
@@ -246,7 +251,12 @@ def _get_handler_of_type(self, type):
         if not self._registry:
             self._registry = {}
 
-            for entry_point in iter_entry_points("sceptre.template_handlers", type):
+            if sys.version_info < (3, 10):
+                entry_points = iter_entry_points("sceptre.template_handlers", type)
+            else:
+                entry_points = iter_entry_points(group="sceptre.template_handlers", name=type)
```

Comment:
Yeah, all this duplication could be dealt with if you defined the iterate_entry_points function I suggested above.

Reply:
done